v/vip/vip-v8.0.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'Tarball'
+
+name = 'vip'
+version = 'v8.0.0'
+
+homepage = 'https://github.com/molgenis/vip/'
+description = """Variant Interpretation Pipeline"""
+
+toolchain = {'name': 'system', 'version': ''}
+
+#https://github.com/molgenis/vip/releases/download/v8.0.0/install.sh
+
+source_urls = [('http://github.com/molgenis/%s/releases/download/%s/' % (name,version))]
+sources = [('install.sh')]
+checksums = ['0ed3d4cae0a518146dc8e26bf4fae4be2d8fee648dda6a07b8fc4bef5a999445']
+
+#
+# set default singularity image folder
+#
+postinstallcmds = [
+	'cd %(installdir)s/ ; cp  %(installdir)s/install.sh ../ ; cd ../ ; mkdir -p /apps/data/vip/resources && mkdir -p /apps/data/vip/images ; ln -sf /apps/data/vip/resources && ln -sf /apps/data/vip/images ; rm -r %(version)s ; bash install.sh -v %(version)s'
+]
+
+sanity_check_paths = {
+    'files': ['install.sh'],
+    'dirs': []
```

Comment:
my first install with another (earlier?) version of your .eb succeeded, but should have failed because `/apps/data/vip/resources` and `/apps/data/vip/images` were not created.

how about adding:
1. these directories to `sanity_check_paths` `dirs`
3. check if `vip.sh` exists
 
can we also check if vip.sh works by running `vip.sh --help`?